content/tutorials/portenta-h7/por-openmv-gs/content.md

```diff
@@ -0,0 +1,211 @@
+
+***In bootloader versions 17 and older there was a bug that could put the Portenta in a boot loop when the transmission aborted while flashing a large firmware file. This was fixed in the bootloader version 18. We strongly advise to update the bootloader before you proceed with the next step of this tutorial!***
+
+After updating the bootloader put the Portenta in bootloader mode by double pressing the reset button on the board. The built-in green LED will start fading in and out. Now open the OpenMV IDE.
+
+![The OpenMV IDE after starting it](assets/por_openmv_open_ide.png)
+
+Click on the "connect" symbol at the bottom of the left toolbar. 
+
+![Click the connect button to attach the Portenta to the OpenMV IDE](assets/por_openmv_click_connect.png)
+
+A pop-up will ask you how you would like to proceed "DFU bootloader(s) found. What would you like to do?". Select "Reset Firmware to Release Version". This will install the latest OpenMV firmware on the Portenta. If it asks you whether it should erase the internal file system you can click "No".
+
+![Install the latest version of the OpenMV firmware](assets/por_openmv_reset_firmware.png)
+
+Portenta's green LED will start flashing while the OpenMV firmware is being uploaded to the board. A terminal window will open which shows you the upload progress. Wait until the green LED stops flashing and fading. You will see a message saying "DFU firmware update complete!" when the process is done.
+
+***Installing the OpenMV firmware will overwrite any existing sketches in the internal flash of Portenta. As a result the M7 port won't be exposed in the Arduino IDE anymore. To re-flash the M7 with an Arduino firmware you need to put the board into bootloader mode. To do so double press the reset button on the Portenta H7 board. The built-in green LED will start fading in and out. In bootloader mode you will see the Portenta M7 port again in the Arduino IDE.***
+
+The Portenta will start flashing its blue LED when it's ready to be connected. After confirming the completion dialog the Portenta should already be connected to the OpenMV IDE, otherwise click the "connect" button once again.
+
+![When the Portenta is successfully connected to the OpenMV IDE a green play button appears in the lower left](assets/por_openmv_board_connected.png)
+
+### 3. Wiring the LED to the Portenta
+
+To be able to change the LED's intensity with PWM we need to first wire the Portenta to the LED. If you want to use other pins than the ones shown here, take a look at the pinout diagram.
+
+![Portenta pinout](assets/Pinout-PortentaH7_latest.png)
+
+Here you can also see what functionality each pin has. In this tutorial wire the pin PC6 via a resistor to the positive leg of the LED, and wire the boards GND to the negative leg of the LED. As shown in the illustration.
+
+[Illustration of a LEDs positive leg on a breadboard wired to the PC6 pin on the Portenta, and the negative end going to the GND pin on the Portenta]()
```

Comment:
Is this an image or a link ?

Reply:
The illustration we talked about should be here.